### DISCUSSION TITLE
SSL certification verification fails in outgoing email

### DISCUSSION THREAD

**Discussion by alexbNz2021** [firefly-iii/firefly-iii#4974] [Firefly III]
Hello 

I'm trying to get this app (docker) to send emails close to pulling out my hair =D

But when I go to send a test email I get this error below 

stream_socket_enable_crypto(): SSL operation failed with code 1. OpenSSL Error messages:
error:1416F086:SSL routines:tls_process_server_certificate:certificate verify failed  

my set up is

Firefly ii (docker) 
mail cow (docker)

Both of them are behind Nginx proxy manager

`MAIL_MAILER=smtp
MAIL_HOST=mail.alexosaurous.co.nz
MAIL_PORT=587
MAIL_FROM=
MAIL_USERNAME=
MAIL_PASSWORD=G
MAIL_ENCRYPTION=tls or SSL tried both`

Thank you for your time

**Comment by JC5** [comment]
Next time a useful title please, so other people can find your problem + solution.
